database/migrations/2022_09_27_091300_run_seeder_for_activity_type.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
```

Comment:
"Blueprint" and "Schema" imports can be removed, as they are not used in this migration.